design/high-level-reactivity.md

```diff
@@ -151,12 +184,22 @@ The rest of the document talks in terms of polling a dependency server for updat
 1. The invalidation server could accept websocket connections, and let the client subscribe to the dependency keys it cares about - this would mean the invalidation is pushed immediately, and then there is one roundtrip to fetch the actual data.
 2. The application server could subscribe to the invalidations, and do the GraphQL queries _on the server_, and then diff that against the current state of the client and send a patch. This would make the system work almost exactly like Meteor today, and could be a good option for applications with fewer users and a great need for low latency.
 
+// I think it's better to go with Just the 1. We could optimize on top of that. It's not a good idea to maintain two different things.
+// If that app is small, it might be okay to change the polling interval smaller. That'll do the trick.
+
 Since both of those approaches don't change anything about the inherent design of the system and are relatively simple to implement, we'll leave them as optimizations for the future.
 
 ### Invalidating dependencies
 
 We haven’t talked at all about how the invalidation server finds out that a dependency version has incremented, and that clients should reload that data. At the lowest level, your code posts to the invalidation server the list of keys that need to be invalidated whenever you make a write to the data store. The next section will talk about a nice high-level wrapper around that for application developers.
 
+// Here, I'll be great we can submit the actual DB changes as well. I mean for simple queries. 
+// Then we could do something with that. Specially in the client.
```

Comment:
There is another person you should talk this. That's chatra. Guys. I will talk to them too.